src/Scouting.elm

```diff
@@ -0,0 +1,65 @@
+module Scouting exposing (Msg, Model, init, update, main, view )
+
+import Browser
+import Html.Attributes exposing (placeholder, value)
+import Html exposing (input , text)
+import Html.Events exposing (onClick , onInput)
+
+
+inputwriter : String -> (String -> Msg) -> String -> Html.Html Msg
+inputwriter placeholderInput msg model =
```

Comment:
You should name this `inputWriter` to follow `camelCase` conventions.
I'd find a more suitable name tough, i.e.`textbox`.